File: samples/csharp/quickstart/chat-with-agent/live-cleanup/live-setup-delete-agent.cs
// Live-validation-only helper: deletes the agent created by quickstart-chat-with-agent.cs
// so repeated live-validation runs don't accumulate agent versions. Not part of the
// documented quickstart narrative.
using Azure.Identity;
using Azure.AI.Projects;
using Azure.AI.Projects.Agents;

// Format: "https://resource_name.services.ai.azure.com/api/projects/project_name"
var foundryProjectEndpoint = "your_project_endpoint";
var foundryAgentName = "your-agent-name";

// Create project client to call Foundry API
AIProjectClient projectClient = new(
    endpoint: new Uri(foundryProjectEndpoint),
    tokenProvider: new DefaultAzureCredential());

projectClient.AgentAdministrationClient.DeleteAgent(foundryAgentName);
Console.WriteLine($"Agent deleted (name: {foundryAgentName})");

File: samples/python/quickstart/create-agent/live_setup_delete_agent.py
# Live-validation-only helper: deletes the agent created by quickstart-create-agent.py
# so repeated live-validation runs don't accumulate agents. Not part of the
# documented quickstart narrative.
from azure.identity import DefaultAzureCredential
from azure.ai.projects import AIProjectClient

# Format: "https://resource_name.services.ai.azure.com/api/projects/project_name"
FOUNDRY_PROJECT_ENDPOINT = "your_project_endpoint"
FOUNDRY_AGENT_NAME = "your-agent-name"

project = AIProjectClient(
    endpoint=FOUNDRY_PROJECT_ENDPOINT,
    credential=DefaultAzureCredential(),
)
project.agents.delete(FOUNDRY_AGENT_NAME)
print(f"Agent deleted (name: {FOUNDRY_AGENT_NAME})")
